### ISSUE TITLE
Not the most accurate.

### ISSUE THREAD

**Issue by 202RaRa** [voteblake/bpm-is#2]
When i saw this i was like wait whats going on, there were no real instructions on how to use it. But once I figured out what was on, I quickly realized it wasn’t that accurate.
I even used my metronome with it to be sure i was tapping with consistency/on beat. But regardless of the tempo i was goin at the tempo seem to rise regardless on the app.

**Comment by 202RaRa** [comment]
If you can get the tempo to be accurate, you on to something great. Keep going.

**Comment by voteblake** [comment]
Thanks for taking a look. Do you remember about what you had the metronome set at and about what the tempo read on the site? Knowing the direction and magnitude of the error might help diagnose it. There are probably practical limitations to accuracy from relying on browser API's for time and click information, but I wouldn't be surprised if there is low-hanging fruit to improve the accuracy from where it is now.

**Comment by 202RaRa** [comment]
No problem. I was kind of excited when i ran accident it. And i had my
metronome in my DAW set to 120 bpm, and the tempo online when be at 120 for
about 8 beats or so and then it would continue to increase going up to
about 150 or so. I hope this feed back helps a lot.

**Comment by voteblake** [comment]
Appreciate it - that definitely sounds wrong. I may not be able to get it 'perfect' but the tempo running away like that isn't even usable. I forgot to ask, which browser was this in? Since I'm relying on browser API's via WASM this behavior might be browser-specific. I'm off work all next week and will be able to poke at this a bit. Knowing the browser might help me reproduce and troubleshoot.

**Comment by 202RaRa** [comment]
Yes exactly like if it at least stuck around the 120-130 that would be a
whole different world. I was using googles chrome browser on my iphone xs
and on my ipad 5th generation same browser on both.
